docs/src/openapi-v1.yaml

```diff
@@ -4504,32 +4516,20 @@ components:
           - TransactionValidityError
           description: Either `DispatchOutcome` if the transaction is valid or `TransactionValidityError` if the result is invalid.
         result:
-          type: string
-          enum:
-          - Ok
-          - CannotLookup
-          - NoUnsignedValidator
-          - Custom(u8)
-          - Call
-          - Payment
-          - Future
-          - Stale
-          - BadProof
-          - AncientBirthBlock
-          - ExhaustsResources
-          - BadMandatory
-          - MandatoryDispatch
-          description: 'If there was an error it will be the cause of the error. If the
-            transaction executed correctly it will be `Ok: []`'
-        validityErrorType:
-          type: string
-          enum:
-          - InvalidTransaction
-          - UnknownTransaction
+          type: object
+          properties:
+            actualWeight:
+              type: string
+              format: unsignedInteger
+              description: The actual weight of the transaction.
+            paysFee:
+              type: string
+              format: boolean
+              description: Whether the transaction pays a fee.
       description: >-
         References:
-        - `UnknownTransaction`: https://crates.parity.io/sp_runtime/transaction_validity/enum.UnknownTransaction.html
-        - `InvalidTransaction`: https://crates.parity.io/sp_runtime/transaction_validity/enum.InvalidTransaction.html
+          - `PostDispatchInfo`: https://docs.rs/frame-support/38.0.0/frame_support/dispatch/struct.PostDispatchInfo.html
+          - `Error Type`: https://paritytech.github.io/polkadot-sdk/master/xcm_runtime_apis/dry_run/enum.Error.html
```

Comment:
We could also get a [DispatchError](https://docs.rs/sp-runtime/39.0.1/sp_runtime/enum.DispatchError.html) no ? Maybe we can add this here along with an additional schema (like `TransactionDispatchOutcome` and `TransactionValidityError` _helper_ schemas) ?